src/precess_cd.jl

```diff
@@ -0,0 +1,78 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _precess_cd{T<:AbstractFloat}(cd::Matrix{T}, epoch1::T, epoch2::T, crval_old::Vector{T},
```

Comment:
For more generality, use `AbstractMatrix` and `AbstractVector` in place of `Matrix` and `Vector` ;-)